### PR TITLE
support GPU for tensorflow inception

### PR DESCRIPTION
**Pull request by cheyang**
**What this PR does / why we need it**:

This PR for 2 things:

1. Make the parameters Cpu, Memory work as described in the Document
2. Support GPU for TensorFlow Inception Model

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by cheyang** [comment]
@viglesiasce Please take a look, thanks.

File: incubator/tensorflow-inception/values.yaml [hunk 1]
```diff
@@ -11,10 +11,6 @@ Replicas: 1
...
-  limits:
-    cpu: 100m
-    memory: 256Mi
-  requests:
-    cpu: 100m
-    memory: 256Mi
\ No newline at end of file
+Cpu: 100m
+Memory: 256Mi
+Gpu: 0
```
Comment (timstoop) [inline incubator/tensorflow-inception/values.yaml]: Although this works fine, it seems that we're moving to standardising on the `resources` variable for this instead, which just takes freestyle yaml so someone is able to set the resources like normal. I think that's cleaner, although adding the cpu option to the docs and/or the values.yaml example might be good as well.

**Review by timstoop — CHANGES_REQUESTED** [review]

**Pull request closed without merging**